CTOR-1043 : Doc(cloud-kubernetes-api) : Update prerequisites for Kubectl version (#3820)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/cloud-kubernetes-api.md

or information about
[secret concept](https://kubernetes.io/docs/concepts/configuration/secret/).

##### Kubectl version 1.24 and above

From  kubectl version 1.24 you need to create a service account with a special annotation:

```
kubernetes.io/service-account.name
```

Manually create a long-lived API token for a ServiceAccount

> Then you create a new Secret with a special annotation, kubernetes.io/service-account.name.

Reviewer: ```suggestion
Then create a new secret with a special annotation, kubernetes.io/service-account.name.
```

> If you want to obtain an API token for a ServiceAccount, you create a new Secret with a special annotation, kubernetes.io/service-account.name.

Author: ```suggestion
Then you create a new Secret with a special annotation, kubernetes.io/service-account.name.
```

```
kubectl apply -f - <<EOF
apiVersion: v1
kind: Secret
metadata:
  name: build-robot-secret
  annotations:
    kubernetes.io/service-account.name: build-robot
type: kubernetes.io/service-account-token
EOF
```

> More information in kubernetes official documentation section [Manually create a long-lived API token for a ServiceAccount](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-long-lived-api-token-for-a-serviceaccount).

Reviewer: ```suggestion
More information in the Kubernetes official documentation: [Manually create a long-lived API token for a Service Account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-long-lived-api-token-for-a-serviceaccount).
```
